main.scss

```diff
@@ -14,67 +14,188 @@ SCSS:
   - timing function: ease-out
 10. Add improvements wherever you see a need for one
 */
-$font-family: Lato, sans-serif;
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+$font-family: Lato, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+  Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji",
+  "Segoe UI Symbol";
+$color-black: #000000;
+$button-color: #eeeeee;
+$background-color: #ffffff;
+$body-background-color: #f5f5f5;
+$font-color: #000000;
+$container-border-color:#eeeeee;
+$font-size : 16px;
+
 
 @mixin not-mobile {
-  @media (min-width: 768px) { @content; }
+  @media (min-width: 768px) {
+    @content;
+  }
 }
 
 @mixin mobile {
-  @media (max-width: 767px) { @content; }
+  @media (max-width: 767px) {
+    @content;
+  }
+}
+
+@mixin transition($property) {
+  transition-duration: .3s;
+  transition-delay: .1s;
```

Comment:
inconsistent - once you are using .1 other time you are adding leading zero (0.5)